apps/portal/astro.config.mjs

```diff
@@ -13,6 +13,6 @@ export default defineConfig({
         ],
       },
     },
-    server: { open: true, port: 3000 },
   },
+  server: { open: true, port: 3000 },
```

Comment:
unrelated actually. i noticed that `yarn dev` was not opening the server on the correct port, because we were passing these options in the wrong place